fe-ionic/sonar-project.properties.template

```diff
@@ -7,8 +7,18 @@ sonar.projectName=@project_id@-@component_id@
 # Comma-separated paths to directories with sources (required)
 sonar.sources=src
 
-# Forced Language (optional)
-sonar.language=js
+# Forced Language: type script
+sonar.language=ts
 
 # Encoding of the source files (optional but recommended as default is ASCII)
 sonar.sourceEncoding=UTF-8
+
+# Exclude test files from coverage analysis
+sonar.coverage.exclusions=**/*.spec.ts
+
+# Exclude non type script files from coverage analysis
+sonar.exclusions=**/*.html,**/*.scss,**/*.json,**/*.ico,**/*.svg
+
+# Define coverage information inside the Angular project
+sonar.javascript.lcov.reportPaths=coverage/@component_id@/lcov.info
```

Comment:
I guess ``@component_id/`` needs to be removed from the path. fe-ionic gets provisioned with the following ``karma.conf.js``: https://github.com/opendevstack/ods-quickstarters/blob/master/fe-ionic/files/karma.conf.js
This one points to the default setting for the ``coverageIstanbulReporter``, which usually is ``coverage/lcov.info``.
Please note: This is different from how it's done in fe-angular. See also https://github.com/opendevstack/ods-quickstarters/pull/306#issuecomment-634749519

Reply:
Thanks Christian 😉